src/toil/statsAndLogging.py

```diff
@@ -173,7 +173,7 @@ def callback(fileHandle):
         # Finish the stats file
         text = json.dumps(dict(total_time=str(time.time() - startTime),
                                total_clock=str(get_total_cpu_time() - startClock)), ensure_ascii=True)
-        jobStore.writeStatsAndLogging(text)
+        jobStore.writeStatsAndLogging(bytes(text, 'utf-8'))
```

Comment:
@DailyDreaming I think this should fix the error we were getting at the end of workflows.

Reply:
That's great.  `writeStatsAndLogging` could probably also benefit from `type hinting` to describe that it takes `bytes` if you could add it!